snakes_cafe.py

```diff
@@ -276,6 +226,105 @@ def optional_menu():
 #             menu[unpacked[i][1]] = {unpacked[i][0]: [0, unpacked[i][1], unpacked[i][2]]}
 #     print(menu)
 
+
+class Order:
+    """
+    new class
+    """
+    def __init__(self):
+        self.default
```

Comment:
Your `__init__()` method should set-up an empty order so that your methods can manipulate the order.